lib/utils.bash

```diff
@@ -80,7 +80,17 @@ get_platform() {
 }
 
 get_arch() {
-  uname -m
+  local platform
+  platform=$(uname -s)
+
+  local arch
+  arch=$(uname -m)
+
+  if [[ "$platform" = "Linux" && "$arch" == "aarch64" ]]; then
+    echo "arm64"
+  else
+    echo "$arch"
+  fi
```

Comment:
Given this applies to Mac too (though [Wikipedia's uname examples](https://en.wikipedia.org/wiki/Uname) suggest `arm64`) we can probably afford to be broad here and avoid an extra platform check:

```suggestion
  local arch
  arch=$(uname -m)

  if [[ "$arch" == "aarch64" ]]; then
    echo "arm64"
  else
    echo "$arch"
  fi
```